cluster/gce/trusty/configure-helper.sh

```diff
@@ -182,6 +184,16 @@ assemble_kubelet_flags() {
   echo "KUBELET_OPTS=\"${KUBELET_CMD_FLAGS}\"" > /etc/default/kubelet
 }
 
+start_kubelet(){
```

Comment:
Yes, the shell functions in this file are called by the upstart jobs in master/yaml and node.yaml. Unfortunately, upstart does not support some syntax in shell, the function keyword is one of them. So I did not add that keyword. In the real gci support https://github.com/kubernetes/kubernetes/tree/master/cluster/gce/gci, we don't have this problem as we use systemd. So the shell scripts there look very close to ContainerVM's script configure-vm.sh